influence_benchmark/experiments/run_experiment.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import multiprocessing as mp
+
+import torch
+
+from influence_benchmark.config.experiment_config import BaseExperimentConfig, ExpertIterationConfig, KTOConfig
+from influence_benchmark.RL.EI import ExpertIteration
+from influence_benchmark.RL.KTO import KTO
+from influence_benchmark.RL.run_EI_iteration import SFT_TRAINING_PATH
+from influence_benchmark.RL.run_KTO_iteration import KTO_TRAINING_PATH
+from influence_benchmark.utils.utils import set_all_seeds
+
+# NOTE: specifying the GPUs here will override the ones in the config file
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Expert Iteration Script")
+    parser.add_argument("--config", type=str, help="Path to the configuration file")
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+
+    config_path = args.config if args.config else DEFAULT_CONFIG_PATH
+    config = BaseExperimentConfig.load(config_path, devices=GPUS)
+
+    if torch.cuda.is_available():
+        print(f"Available CUDA devices: {torch.cuda.device_count()}")
+    else:
+        print("CUDA is not available.")
+
+    if config.seed is not None:
+        print(f"Setting all seeds to: {config.seed}")
+        set_all_seeds(config.seed)
+
+    mp.set_start_method("spawn", force=True)
+
+    print(f"Total of {config.num_envs_per_device * len(config.devices)} parallel envs")
+
+    experiment_class = None
+    training_script_path = None
+    if isinstance(config, ExpertIterationConfig):
+        experiment_class = ExpertIteration
+        training_script_path = SFT_TRAINING_PATH
+    elif isinstance(config, KTOConfig):
+        experiment_class = KTO
+        training_script_path = KTO_TRAINING_PATH
+    else:
+        raise ValueError(f"Unknown experiment type: {type(config)}")
+
+    experiment = experiment_class(
```

Comment:
this doesn't include override_initial_traj_path

Reply:
Whoops, my bad